recipe/meta.yaml

```diff
@@ -11,30 +11,35 @@ source:
 
 build:
   number: 0
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - optuna = optuna.cli:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
     - alembic >=1.5.0
     - cmaes >=0.9.1
     - colorlog
     - numpy
     - packaging >=20.0
-    - python >=3.7
+    - python
+    - PyYAML
```

Comment:
I like all lower case for consistency....not sure if we have policies for that though

Reply:
Fixed